### PR TITLE
Feature/27 composer 2 fix

### PR DESCRIPTION
**Pull request by finnlewis**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,18 +11,18 @@
...
+        "composer/installers": "^1.9",
         "cweagans/composer-patches": "^1.6",
-        "drupal/core-composer-scaffold": "^8.8",
-        "drupal/core-project-message": "^8.8",
-        "drupal/core-recommended": "^8.8",
-        "localgovdrupal/localgov": "^1.0@alpha"
+        "drupal/core-composer-scaffold": "^8.9",
+        "drupal/core-project-message": "^8.9",
+        "drupal/core-recommended": "^8.9",
+        "localgovdrupal/localgov": "dev-feature/152-composer-2-fix"
```
Comment (Adnan-cds) [inline composer.json:19]: I am just wondering if you want to first make a new alpha release of localgovdrupal/localgov.  That should allow us to stick to **^1.0@alpha ** for localgovdrupal/localgov.

Comment (Adnan-cds) [reply composer.json:19]: Rest of the changes look fine to me.

Comment (finnlewis) [reply composer.json:19]: Yes - thanks @Adnan-cds  - that was a temporary change to allow testing these two together. I'll merge the other PR and tag a release, then switch this back... and who knows, tests might pass!

**Comment by finnlewis** [comment]
@Adnan-cds there are no CI tests running on this repo. 
Is that by intention? 
Shall we add them or you happy to merge this and test the whole lot locally against composer2?

**Comment by Adnan-cds** [comment]
> Is that by intention?

I don't know :)

> Shall we add them or you happy to merge this and test the whole lot locally against composer2?

Let me try locally and get back to you shortly.

**Review by Adnan-cds — APPROVED** [review]
Works fine for me when I tried the following:
```
$ ./composer --version
Composer version 2.0.4 2020-10-30 22:39:11
$ ./composer create-project --stability dev --no-cache localgovdrupal/localgov-project:dev-feature/27-composer-2-fix
```

Thanks :)

**Pull request merged**